src/scene/text/canvas/CanvasTextSystem.ts

```diff
@@ -58,15 +68,39 @@ export class CanvasTextSystem implements System
         return { width, height };
     }
 
-    public getTexture(text: string, resolution: number, style: TextStyle, textKey: string)
+    /**
+     * This is a function that will create a texture from a text string, style and resolution.
+     * Useful if you want to make a texture of your text and use if for various other pixi things!
+     * @param options - The options of the text that will be used to generate the texture.
+     * @param options.text - the text to render
+     * @param options.style - the style of the text
+     * @param options.resolution - the resolution of the texture
+     * @returns the newly created texture
+     */
+    public getTexture(options: TextOptions): Texture
```

Comment:
Can you deprecate the old function signature

Reply:
done! :D